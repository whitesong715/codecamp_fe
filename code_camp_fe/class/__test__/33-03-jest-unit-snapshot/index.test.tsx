import JestUnitTest from "../../pages/55sec/33-03-jest-unit-snapshot"
import { render, screen } from '@testing-library/react'
import "@testing-library/jest-dom"

// 기존과 바뀐점을 찾고, 없으면 snapshot을 남긴다(사진을 찍는다)
it("기존 사진과 바뀐것이 없는지 비교 - 스냅샷 테스트", () => {
    const result = render(<JestUnitTest />)

    expect(result.container).toMatchSnapshot()
})