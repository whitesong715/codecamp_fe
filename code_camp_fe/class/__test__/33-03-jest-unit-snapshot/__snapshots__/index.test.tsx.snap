// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`기존 사진과 바뀐것이 없는지 비교 - 스냅샷 테스트 1`] = `
<div>
  <div>
    철수는 13살
  </div>
  철수의 취미 입력하기: 
  <input
    type="text"
  />
  <button>
    놀러가기
  </button>
</div>
`;
